BlinkIDSample/BlinkID-ComposeFragmentSample/src/main/java/com/microblink/blinkid/MainActivity.kt

```diff
@@ -0,0 +1,330 @@
+                    Toast.LENGTH_SHORT
+                ).show()
+            }
+
+            supportFragmentManager.beginTransaction().apply {
+                mScanOverlay = null
+                showScanning.value = 2
+            }.commit()
+
+        }
+
+        override fun onUnrecoverableError(throwable: Throwable) {}
+    }
+
+    private fun createOverlayController(
+        context: Context,
+        recognizerBundle: RecognizerBundle,
+        scanResultListener: ScanResultListener,
+    ): BlinkIdOverlayController {
+        val overlaySettings = BlinkIdOverlaySettings.Builder(recognizerBundle).build()
+        val overlayStrings = ReticleOverlayStrings.Builder(context).build()
+        val customStyle = R.style.CustomStyle
+        val reticleOverlay: BlinkIdOverlayView =
+            ReticleOverlayView(true, true, true, false, true, true, overlayStrings, customStyle)
+        return BlinkIdOverlayController(overlaySettings, scanResultListener, reticleOverlay)
+    }
+
+    // can be used just like the previous function
+    private fun createOverlayController(
+        recognizerBundle: RecognizerBundle,
+        scanResultListener: ScanResultListener
+    ): BlinkIdOverlayController {
+        val overlayStrings = ReticleOverlayStrings.Builder(this@MainActivity).build()
+        val settings = BlinkIdUISettings(recognizerBundle).also {
+            it.setShowFlashlightWarning(true)
+            it.setShowIntroductionDialog(true)
+            it.setShowOnboardingInfo(true)
+            it.setShowMandatoryFieldsMissing(true)
+            it.setAllowHapticFeedback(true)
+            it.setStrings(overlayStrings)
+            it.setOverlayViewStyle(R.style.CustomStyle)
+        }
+        return settings.createOverlayController(this, scanResultListener)
+    }
+
+    @Composable
+    fun MainMenu() {
+        Column(
+            Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            val mButtonColors: ButtonColors = ButtonDefaults.buttonColors(
+                containerColor = colorResource(id = R.color.mb_lime), contentColor = colorResource(
+                    id = R.color.mb_blue_deep
+                )
+            )
+            Button(colors = mButtonColors, onClick = {
+                supportActionBar?.hide()
+                mRecognizerBundle = RecognizerBundle(BlinkIdSingleSideRecognizer())
+                showScanning.value = 1
+            }) {
+                Text("SingleSide Scan")
+            }
+            Button(colors = mButtonColors, onClick = {
+                supportActionBar?.hide()
+                mRecognizerBundle = RecognizerBundle(BlinkIdMultiSideRecognizer())
+                showScanning.value = 1
+            }) {
+                Text("MultiSide Scan")
+            }
+
+        }
+    }
+
+    @Composable
+    fun ResultScreen() {
+        val result =
```

Comment:
Drzis gore refrencu na RecognizerBundle, ne trebas ga izvlaciti iz RecognizerRunnerFragmenta:
`        val result = mRecognizerBundle.recognizers[0]
`
Vidim da si to na drugim mjestima koristio
I preimenuj varijablu na recognizer ili nest drugo jer to nije result